src/index.ts

```diff
@@ -1479,14 +1479,15 @@ function FlatpickrInstance(
 
       if (lostFocus && isIgnored) {
         if (self.config.allowInput) {
-          self.setDate(self._input.value, 
-            true, 
-            eventTarget === self.altInput
```

Comment:
`eventTarget` never becomes `self.altInput` here, because as described in the method name this is the event handler for `documentClick`.